src/Components/OATHeader/OATHeader.tsx

```diff
@@ -141,13 +153,26 @@ const OATHeader = ({ elements, dispatch, state }: OATHeaderProps) => {
                         items.push(newItem.content);
                     } else {
                         allValidFiles = false;
+                        filesErrors.push(validJson);
                     }
                 } catch (error) {
-                    console.log(error);
-                    alert(`Issue on file ${current.name} \r ${error}`);
                     allValidFiles = false;
                 }
             }
+            if (filesErrors.length > 0) {
```

Comment:
Another way to describe this would be `!allValidFiles`. If we did something like that, then we could do an `if / else` here.